.github/workflows/deploy-swagger.yml

```
@@ -0,0 +1,48 @@
+      - name: Copy Swagger UI template
+        run: |
+          curl https://raw.githubusercontent.com/swagger-api/swagger-ui/master/dist/index.html -o docs/index.html
+          sed -i 's|https://petstore.swagger.io/v2/swagger.json|swagger.json|' docs/index.html
+
+      - name: Commit & Push docs
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'actions@github.com'
+          git add docs
+          git commit -m "Update Swagger UI"
+          git push
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)